pkg/utils/utils.go

```diff
@@ -36,7 +36,6 @@ const (
 	ClusterTypeOpenshift  = "openshift"
 	ClusterTypeKubernetes = "kubernetes"
 	VendorMellanox        = "15b3"
```

Comment:
i think, with this change, `VendorMellanox` is also no longer used.

Reply:
It's still used here https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/235/files#diff-81ddbadfb415ccbb9c7af84f11668d1aa5e53c34025bf86d4702f16b4e42f045R708

Reply:
ah, being used here: https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/c400742d910d731dfda9e81475d3b8047f051485/pkg/utils/utils.go#L714

search in github is not very good :(

Thanks for pointing this out